### PR TITLE
use new_modals only for pecan cookies

### PR DESCRIPTION
**Pull request by javisantana**

### CODE REVIEW

File: app/views/admin/visualizations/show.html.erb [hunk 1]
```diff
@@ -6,11 +6,7 @@
   <% end %>
 
   <% content_for(:css) do %>
-    <% if current_user.has_feature_flag?('new_modals') %>
-      <%= stylesheet_link_tag 'common.css', 'editor.css', 'cdb.css', 'map.css', :media => 'all' %>
-    <% else  %>
-      <%= stylesheet_link_tag 'old_common.css', 'table.css', 'cdb.css', 'map.css', :media => 'all' %>
```
Comment (xavijam) [inline app/views/admin/visualizations/show.html.erb]: y table??

**Comment by javierarce** [comment]
Thanks, added in issue-4054 (adding table.css, too)

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5144/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5145/))

**Pull request closed without merging**